lib/widgets/form_builder.dart

```diff
@@ -29,6 +37,13 @@ class _FormBuilderState extends State<FormBuilder> {
     setState(() {});
   }
 
+  String? lastSuggestedTitle = DateTime.now().toIso8601String();
+  void onLabelChange(String? suggestedTitle) {
```

Comment:
having same name is confusing, I know the other one would need `widget.` but still should at least be private then